packages/gatsby-source-kontent/src/types.d.ts

```diff
@@ -15,6 +15,10 @@ interface CustomPluginOptions extends PluginOptions {
   includeTaxonomies: boolean = false;
   authorizationKey: string = null;

```

Comment:
I'd recommend revisiting the concept of storing auth key into options object. It guides you to include your token in the src control which is not correct. We should introduce and use practice using .evn, shouldn't we?

Reply:
yeap - will adjust readme :)

Reply:
Fixed in 7612ac7ea38f6db9bd9b3e2cda71762c21a2288e

Reply:
just one minor thing - update yarn lock as well